mvp-gate-party/src/main/kotlin/cz/tyckouni/mvpgate/party/business/sp/SpService.kt

```diff
@@ -2,5 +2,13 @@ package cz.tyckouni.mvpgate.party.business.sp
 
 import cz.tyckouni.mvpgate.party.business.Service
 import cz.tyckouni.mvpgate.party.persistence.sp.Sp
+import java.util.*
 
-interface SpService : Service<Sp>
+interface SpService : Service<Sp> {
+
+    fun findByName(name: String): List<Sp>
+
+    fun create(name: String): Sp
```

Comment:
In future, I think we should introduce here some new type as a input. Something like `NewSp`. But this is sufficient for now.